snap_ci/prod_clean.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# bail out on errors
+set -e
+set -x
+
+# declare function for EXIT trap
+function cleanup {
+    echo "Destroying droplet..."
+    vagrant destroy /prod/ -f
+}
+
+# Ensure that DigitalOcean droplet will be cleaned up
+# even if script errors (e.g., if serverspec tests fail).
+trap cleanup EXIT
+
+# If the previous build in snap-ci failed, the droplet
+# will still exist. Ensure that it's gone with a pre-emptive destroy.
+cleanup
+
+# Find the root of the git repository. A simpler implementation
+# would be `git rev-parse --show-toplevel`, but that must be run
+# from inside the git repository, whereas the solution below is
+# directory agnostic. Exporting this variable doesn't work in snapci,
+# so it must be rerun in each stage.
+repo_root=$( dirname "$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd )" )
+
+# Copy staging vars for use as prod vars; will need to skip "validate" role.
+sed -r -e 's/(app|mon)-staging/\1-prod/' \
+    "${repo_root}"/install_files/ansible-base/staging-specific.yml > \
+    "${repo_root}"/install_files/ansible-base/prod-specific.yml
+
+# Skip "validate" so reused vars don't cause failure,
+# and skip "grsec" because DigitalOcean hosts don't support custom kernels.
+export SECUREDROP_PROD_SKIP_TAGS=validate,grsec
+
+# Assume that /prod/ instances will be created from scratch, as part
+# of a full provisioning run, so initial SSH connections must be
+# made directly, rather than over Tor.
+unset SECUREDROP_SSH_OVER_TOR
+
+# Up the host in a separate command to avoid snap-ci command timeouts.
```

Comment:
Preventing timeouts in snapci is now down in the snap config itself, rather than inside these bash scripts. It's still necessary to run the `up` and `provision` commands separately, and I've updated the associated comments in efcb1c9 to explain.